### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [tenacityteam/tenacity-legacy#718] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by fossdd** [comment]
This looks interesting. However, it's easier if someone else (outside of the tenacity team) would maintain it, as we will (hopefully) have a offical way to install Tenacity (plus the AUR). Maintaining more sources makes it more difficult.

**Comment by fitojb** [comment]
Thanks for sharing, I didn’t know about Pacstall!

**Comment by AnotherFoxGuy** [comment]
I think it would be easier to add tenacity to the mpr, since we can reuse most of the aur pkgbuild for it
https://mpr.makedeb.org

**Comment by Elsie19** [comment]
Pacstall already has [tenacity](https://github.com/pacstall/pacstall-programs/blob/master/packages/tenacity-git/tenacity-git.pacscript)!
